bin/utils/bulk_compression.py

```diff
@@ -0,0 +1,418 @@
+# PCSX2 - PS2 Emulator for PCs
```

Comment:
```suggestion
#!/usr/bin/env python3
```
Add a shebang to the very top of the line so that us Linux folks can just run the script directly with `./bulk_compression.py` instead of typing out the whole command (`python3 bulk_compression.py`

Reply:
Done.